wrapanapi/systems/rhevm.py

```diff
@@ -860,6 +860,27 @@ def list_cluster(self, **kwargs):
         cluster_list = self.api.system_service().clusters_service().list(**kwargs)
         return [cluster.name for cluster in cluster_list]
 
+    def list_disks(self, **kwargs):
+        disks_list = self.api.system_service().disks_service().list()
+        if kwargs:
+            disk_status = kwargs.get('status')
+            if disk_status:
+                disk_status = disk_status.upper()
```

Comment:
Think we should raise ValueError here against `list(types.DiskStatus)`. Since its an enum, it'll list its enumerations.

https://docs.python.org/3/library/enum.html#iteration  (py2.7 too)

Reply:
Actually it appears to be much simpler to use `__members__` to achieve this:

```
kwargs.get('status').upper() in types.DiskStatus.__members__
```